### PR TITLE
Add Sentiment area under Comparison Analysis

### PR DESCRIPTION
**Pull request by DidierRLopes**
The new commands allow:
Captura de ecrã 2021-03-20, às 15 57 01

Captura de ecrã 2021-03-20, às 15 57 06

Also, fixed a bug that was adding the main ticker to the list of similar companies when selecting one of the financials command.

### CODE REVIEW

File: gamestonk_terminal/comparison_analysis/README.md [hunk 1]
```diff
@@ -52,13 +56,13 @@ Historical price comparison between similar companies [Source: Yahoo Finance]
 
 ![historical](https://user-images.githubusercontent.com/25267873/110699590-ef2b8500-81e6-11eb-95e3-144793a83a80.png)
 
-## corr <a name="corr"></a>
+## hcorr <a name="hcorr"></a>
 
 ```text
 usage: corr [-s L_SIMILAR] [-a L_ALSO] [-t TYPE_CANDLE]
 ```
```
Comment (jmaslek) [inline gamestonk_terminal/comparison_analysis/README.md:63]: usage: hcorr

File: gamestonk_terminal/comparison_analysis/README.md [hunk 2]
```diff
@@ -114,4 +118,30 @@ Cashflow financials comparison between similar companies [Source: Market Watch]
...
+* -s : similar companies to compare with. Default pre-loaded ones.
+* -a : apart from loaded similar companies also compare with.
+
+<img width="754" alt="Captura de ecrã 2021-03-20, às 15 57 06" src="https://user-images.githubusercontent.com/25267873/111920503-45ba7e00-8a87-11eb-9a5c-aefb20793f7f.png">
+
+## scorr <a name="scorr"></a>
+
+```text
+usage: corr [-s L_SIMILAR] [-a L_ALSO] [-t TYPE_CANDLE]
+```
```
Comment (jmaslek) [inline gamestonk_terminal/comparison_analysis/README.md:138]: `usage: scorr`  ?

**Review by jmaslek — CHANGES_REQUESTED** [review]
#ilikethecorrelations

File: gamestonk_terminal/comparison_analysis/README.md (hunk 2 above)
Comment (DidierRLopes) [reply gamestonk_terminal/comparison_analysis/README.md]: Nice, thanks! Should be fixed now

**Comment by jmaslek** [comment]
Another comment:  when I go to ca menu, the only options I see are get/select.  Not sure if you want more options at first (note I don't have polygon API).

I see the menu pop up after I do select/ but if I load AAPL, then I can just use (ie) `hcorr -s msft,amzn` without needing the select first (but that is not an option in help until I do select).

Either way  I think this can be merged.

**Comment by DidierRLopes** [comment]
> Another comment: when I go to ca menu, the only options I see are get/select. Not sure if you want more options at first (note I don't have polygon API).
> 
> I see the menu pop up after I do select/ but if I load AAPL, then I can just use (ie) `hcorr -s msft,amzn` without needing the select first (but that is not an option in help until I do select).
> 
> Either way I think this can be merged.

Good call, I agree with that. I'll display all menu options on that basis!

The Polygon API is just if you want to use the "get" command, that gives similar companies.

**Comment by jmaslek** [comment]
looks good to me!

**Pull request merged**